src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,11 +4,15 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.info("completeOrder method was called. Params: userId = {}", userId);
```

Comment:
```suggestion
        logger.debug("completeOrder method was called. Params: userId = {}", userId);
```
same as in Auth Service Impl